addons/binding/org.openhab.binding.freebox/META-INF/MANIFEST.MF

```diff
@@ -23,14 +23,18 @@ Import-Package: com.google.common.base,
  org.apache.commons.logging,
  org.eclipse.smarthome.config.core,
  org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.audio,
  org.eclipse.smarthome.core.common.registry,
  org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.net,
  org.eclipse.smarthome.core.thing,
  org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.util,
  org.eclipse.smarthome.core.types,
  org.eclipse.smarthome.io.net.http,
  org.eclipse.smarthome.io.transport.mdns.discovery,
  org.osgi.framework,
+ org.osgi.service.component;version="1.2.2",
```

Comment:
Sorry I think I missed this in my earlier reviews, you should not mention a version here.

Please give kai a ping of you picked it up and commit it separate (github can do the squashing) it eases the review.